### PR TITLE
Code improvements

### PR DESCRIPTION
**Pull request by mkangia**
Just generally things noticed during code walkthrough

### CODE REVIEW

**Review by millerdev — COMMENTED** [review]
Most of the changes in this PR seem unnecessary with the possible exception of https://github.com/dimagi/commcare-hq/pull/27769/commits/3c639674b26ea943d2575c5efe740c4977319e9f. However, I'm glad you're walking through and learning the SMS code! I also find it useful to make edits like this when I'm learning about code. Most of the time I do it on a private branch and then delete the branch when I'm done since most of the benefit was for my learning.

File: corehq/messaging/scheduling/models/abstract.py [hunk 1]
```diff
@@ -493,10 +493,10 @@ def send_sms_message(self, domain, recipient, phone_entry_or_number, message, lo
 
         if isinstance(phone_entry_or_number, PhoneNumber):
             send_sms_to_verified_number(phone_entry_or_number, message, metadata=metadata,
-                logged_subevent=logged_subevent)
+                                        logged_subevent=logged_subevent)
```
[the remark this answers is not in the document]
Comment (mkangia) [reply corehq/messaging/scheduling/models/abstract.py]: pyCharm was complaining about this indentation and I have been always following the change i made. Good to know that both are valid.

> The disadvantage of minor stylistic changes like this is that it obscures file history

Agreed. There were just a lot of them so I could not resist but knowing now that both are valid, I am not going to look at them as incorrect.

File: corehq/apps/sms/api.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def get_sms_class():
     return QueuedSMS if settings.SMS_QUEUE_ENABLED else SMS
 
 
-def send_sms(domain, contact, phone_number, text, metadata=None, logged_subevent=None):
+def send_sms(domain, contact, phone_number, text, metadata=None, logged_sub_event=None):
```
[the remark this answers is not in the document]
Comment (mkangia) [reply corehq/apps/sms/api.py:127]: I did some back and forth on the risks of this one and then left it to a second opinion on the PR. I was just targetting that specific name only and not "subevent" but i see the discrepancy it introduces now.

**Comment by mkangia** [comment]
Closed in favour of https://github.com/dimagi/commcare-hq/pull/27774

Let me know if you think https://github.com/dimagi/commcare-hq/pull/27769/commits/f41c64be08ea22b49d2092ac09b4750cd3157aaf is worth including in.
Changes just the method signature so should not harm the history look up.

**Comment by mkangia** [comment]
> Most of the time I do it on a private branch and then delete the branch when I'm done since most of the benefit was for my learning.

that is true and great advice. I do that as well when I am just putting comments for my own learning or trying out something just as a challenge/fun

File: corehq/messaging/scheduling/models/abstract.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (millerdev) [reply corehq/messaging/scheduling/models/abstract.py]: Might be worth turning off that style rule in pyCharm so you don't have to suffer the annoyance of its complaining all the time.

**Comment by millerdev** [comment]
> Let me know if you think f41c64b is worth including in.

I typically do not use `staticmethod` unless there is some code that calls the function as `Class.method(arg)`, so I wouldn't bother.

File: corehq/messaging/scheduling/models/abstract.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkangia) [reply corehq/messaging/scheduling/models/abstract.py]: Was curious about this and so searched a bit to find https://www.python.org/dev/peps/pep-0008/#code-lay-out
Is that a good standard to follow?

```
# Correct:

# Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                                              var_three, var_four)

# Wrong:

# Arguments on first line forbidden when not using vertical alignment.
foo = long_function_name(var_one, var_two,
    var_three, var_four)
```
I do understand the point for history keeping but just want to be more aware for new code that i add.

**Pull request closed without merging**